mock-interactions.js

```diff
@@ -40,7 +40,10 @@
       bubbles: true,
       cancelable: true,
       clientX: xy.x,
-      clientY: xy.y
+      clientY: xy.y,
+      // Make this a primary input.
+      buttons: 1, // http://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons
+      which: 2    // http://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/which
```

Comment:
This won't get copied to the event, because `which` is super-double-secret deprecated.